antora/docs/modules/ROOT/templates/release_policy.hbs

```diff
@@ -4,6 +4,23 @@
 
 These rules are applied to pipeline run attestations associated with container images built by HACBS
 
+== Available rule collections
+
+[cols="2,6"]
+|===
+|*Name*
+|*Description*
+
+{{#each releaseCollections }}
+| `{{ info.title }}`
+a| {{ info.description }}
```

Comment:
Is the leading `a` character here expected?

Reply:
Yeah, it's an attribute to [interpret the cell content as Asciidoc](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator)